packages/mobile/src/transactions/feed/TransferFeedItem.test.tsx

```diff
@@ -328,6 +328,48 @@ describe('TransferFeedItem', () => {
     })
   })
 
+  it('renders correctly for sent invites', async () => {
+    const contactPhoneNumber = '+14155553695'
+    const contactName = 'Kate Bell'
+    const { getByTestId } = renderScreen({
+      type: TokenTransactionTypeV2.InviteSent,
+      storeOverrides: {
+        transactions: {
+          inviteTransactions: {
+            [MOCK_TX_HASH]: {
+              recipientIdentifier:
+                '0xd2c326a68d07b060be189bbe05a9abee8f10573a23cd448122d2efd122b4e05a',
+              paymentId: '0x72a8898b40bD3CAf7e63664b61dBBa2E98fC123D',
+            },
+          },
+        },
+        recipients: {
+          phoneRecipientCache: {
+            [contactPhoneNumber]: {
+              e164PhoneNumber: contactPhoneNumber,
+              name: contactName,
+            },
+          },
+        },
+        identity: {
+          e164NumberToSalt: {
+            [contactPhoneNumber]: 'hmq1hMZ08BYOg',
+          },
+        },
+      },
+    })
+
+    expectDisplay({
+      getByTestId,
+      expectedTitleSections: [
+        `feedItemEscrowSentTitle, {"context":null,"nameOrNumber":"${contactName}"}`,
```

Comment:
nit: it's better to have hardcoded expectations to minimize room for errors:
```suggestion
        `feedItemEscrowSentTitle, {"context":null,"nameOrNumber":"Kate Bell"}`,
```

Reply:
fair enough, I updated!